canary/environments/auth-with-email/package.json

```diff
@@ -3,6 +3,6 @@
   "name": "auth-with-email-environment",
   "version": "0.0.1",
   "scripts": {
-    "pull": "amplify pull --appId d39wm44u5j8yn5 --envName staging"
+    "pull": "amplify pull --appId d38yznh7fio4c0 --envName staging"
```

Comment:
This is the same backend from https://github.com/aws-amplify/amplify-ui/pull/2596.